sites/svelte.dev/src/lib/search/stores.js

```diff
@@ -0,0 +1,8 @@
+import { writable } from 'svelte/store';
+import { persisted } from 'svelte-local-storage-store';
```

Comment:
Can we use the function from `kit.svelte.dev` instead of the package?

Reply:
I was thinking of moving kit's function to this as well, when I move this to `site-kit`. Reason: Library is super tiny, and it syncs localstorage as well, so tab synchronisation is also there. It's a nice to have. If you really insist I'll copy paste that function back here

Reply:
Sync is a good point, haven't thought of that. Ok with keeping it then.